src/index.ts

```diff
@@ -9,7 +9,7 @@ declare global {
 
 export interface ContextEntry {
   readonly key: string
-  readonly type: Any | NeverType
+  readonly type: Decoder<any, any>
```

Comment:
@gcanti do we loose anything by dropping 'never' ?

Reply:
@sledorze `| NeverType` was added for variance issues, IMO this is an actual improvement due to the Decoder/Encoder splitting

Reply:
@gcanti  ok good to know 👍

Reply:
For reference, these are the lines that currently cause variance issues if I remove `| NeverType`

https://github.com/gcanti/io-ts/blob/37c74a5e2038de063a950f9ba8d18b1f132ef450/src/index.ts#L756
https://github.com/gcanti/io-ts/blob/37c74a5e2038de063a950f9ba8d18b1f132ef450/src/index.ts#L875